Include/internal/pycore_pyerrors.h

```diff
@@ -114,6 +108,10 @@ PyAPI_FUNC(void) _Py_NO_RETURN _Py_FatalRefcountErrorFunc(
 
 #define _Py_FatalRefcountError(message) _Py_FatalRefcountErrorFunc(__func__, message)
 
+
+/* number of items stored in the stack for each exception */
+#define PY_EXC_INFO_STACK_SIZE 1
```

Comment:
Is this necessary now?
I don't see why we would ever want to increase it, and I don't see how we could ever decrease it.

Reply:
The name is probably weird now, but I think it helps readability to have a macro. If we remove this we should find another way to make the code (particularly in compile.c) clear about where exceptions are showing up. Figuring out where to use this macro was the hardest part of making this change (because when you get it wrong it just segfaults before you finish make regen-all).

Reply:
On second thought there is no point in a macro for this unless we all agree on some kind of convention because it probably won't be updated correctly when people make changes. So I'll remove it.